runtime/runtime.go

```diff
@@ -2984,7 +3029,7 @@ func (r *interpreterRuntime) newAccountKeysGetFunction(
 					inter,
 					invocation.GetLocationRange,
 					accountKey,
-					inter.PublicKeyValidationHandler,
+					alwaysValidates, // key from FVM has already been validated
```

Comment:
@tarakby Can we assume that the FVM always returns keys that are both validated *and* valid?

Reply:
Just to make sure I'm answering the right question, this PR is about the raw `PublicKey` structure, while the function above is about an account public key (wrapper around a raw public key). 

But in general, yes data coming from the FVM is validated and valid.